### ISSUE TITLE
Underscore missing in Validator::fullyQualifiedDomainName

### ISSUE THREAD

**Issue by MikeAT** [Badcow/DNS#65]
Hi,

When parsing this (valid and working) record 

`testnaptr.at. 30 IN NAPTR 100 50 "s" "SIP+D2U" "" _sip2._udp.testnaptr.at.`

I got this exceptions

```
Could not extract Rdata from resource record "testnaptr.at. 30 IN NAPTR 100 50 "s" "SIP+D2U" "" _sip2._udp.testnaptr.at.".
Replacement must be a fully qualified domain name. "_sip2._udp.testnaptr.at." given
```

The replacement is a valid FQDN, but the Validator::fullyQualifiedDomainName method is missing the underscore in the regexp.

Best,
Michael

**Comment by samuelwilliams** [comment]
I think I might need to build a different validator method. "_sip2._udp.testnaptr.at." is not a valid FQDN - but it _is_ valid hostname. I'm gonna have to do a bit more research.
The other option is I make the validation a bit more liberal.

**Comment by samuelwilliams** [comment]
Hi @MikeAT ,
[Checkout this branch](https://github.com/Badcow/DNS/tree/issue-65) and let me know if this fixes your issue.

**Comment by MikeAT** [comment]
Hi,

works as expected, thanks you very much for the quick fix.

Best,
Michael

**Comment by samuelwilliams** [comment]
Fixed in release 3.4.1
